baseplate/experiments/providers/r2.py

```diff
@@ -57,6 +57,7 @@ def max_variant_size(variant_size, num_variants):
           created or when an LoID cookie was generated.
     """
 
+    # pylint: disable=redefined-builtin
```

Comment:
Can we just ignore this one globally?  I'm not sure how useful it is.

Reply:
I'm sorta ambivalent, but the example given here does seem somewhat like it'd be pretty annoying to run into: https://docs.quantifiedcode.com/python-anti-patterns/correctness/assigning_to_builtin.html

(I just wish they hadn't claimed the name `id` for themselves. Grr.)